teaching/index.md

```diff
@@ -11,6 +11,12 @@ Julia is ready for the classroom. We encourage instructors to participate in the
 
 Julia is now being used in several university courses. If you know of other classes using Julia for teaching, please [submit a pull request](https://github.com/JuliaLang/julialang.github.com/pulls) to update this list.
 
+- University of Edinburgh, Spring 2016
+  * [MATH11146](http://www.drps.ed.ac.uk/15-16/dpt/cxmath11146.htm), Modern optimization methods for big data problems (Prof. [Peter Richtarik](http://www.maths.ed.ac.uk/~prichtar/index.html))
+
+- Universit\'{e} Paul Sabatier, Toulouse, Fall 2015
```

Comment:
OK, got it.

On Wed, Sep 16, 2015 at 3:31 PM, Stefan Karpinski <notifications@github.com>
wrote:

> In teaching/index.md
> <https://github.com/JuliaLang/julialang.github.com/pull/267#discussion_r39628897>
> :
>
> > @@ -11,6 +11,12 @@ Julia is ready for the classroom. We encourage instructors to participate in the
> >
> >  Julia is now being used in several university courses. If you know of other classes using Julia for teaching, please [submit a pull request](https://github.com/JuliaLang/julialang.github.com/pulls) to update this list.
> >
> > +- University of Edinburgh, Spring 2016
> > +  * [MATH11146](http://www.drps.ed.ac.uk/15-16/dpt/cxmath11146.htm), Modern optimization methods for big data problems (Prof. [Peter Richtarik](http://www.maths.ed.ac.uk/~prichtar/index.html))
> > +
> > +- Universit\'{e} Paul Sabatier, Toulouse, Fall 2015
>
> Markdown doesn't support this LaTeXism; you can use Unicode characters
> though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaLang/julialang.github.com/pull/267/files#r39628897>
> .
>